src/components/pages/document/list/view/index.tsx

```diff
@@ -0,0 +1,10 @@
+import useObservationFilter from "@components/pages/document/common/use-document-filter";
```

Comment:
This is also unnecessary since on document list UI we are not going to have grid view